base/interactiveutil.jl

```diff
@@ -64,7 +64,27 @@ function edit(path::AbstractString, line::Integer=0)
     nothing
 end
 
-edit(f)          = edit(functionloc(f)...)
+function edit(f)
+    ms = methods(f)
+    nm = length(ms)
+    if nm <= 1
+        edit(functionloc(f)...)
+    else
+        for (i, m) in enumerate(ms)
+            @printf("%4u: %s\n", i, m)
+        end
```

Comment:
Should this be limited in some way? Some functions contain allot of methods (e.g. `methods(+)`)